src/views/Calendar.vue

```diff
@@ -37,25 +52,114 @@ interface CalendarEvent {
     allEvents: {
       query: AllEvents,
     },
+    allUsers: {
+      query: AllUsersBirthdays,
+    },
+    allCategories: {
+      query: AllCategories,
+    },
+  },
+  computed: {
+    reversedMessage() {
+      // `this` points to the vm instance
+      return this.$data.split('').reverse().join('');
```

Comment:
בבקשה תסבירי כי לא כל כך מובן מה השורה הזאת עושה, אם חשוב אפשר לעשות אותו בתוך המחלקה ולא בתוך האנוטציה

Reply:
I forgot remove it